### DISCUSSION TITLE
How to update to latest release

### DISCUSSION THREAD

**Discussion by joevk5ei** [merbanan/rtl_433#2496] [General]
What is the process of updating my current version of rtl_433 to the latest release version ?   I am using Ubuntu 20.04

**Comment by gdt** [comment]
1) Look in the source code for a file that might explain how to build form source.
2) Read https://github.com/merbanan/rtl_433/blob/master/docs/BUILDING.md

**Reply by joevk5ei** [reply]
Ok, I understand, a rebuild from latest releases is required.

**Comment by AutomGuy** [comment]
Hello @joevk5ei ,
at Ubuntu 20.04 is rtl_433 already installed from the distro. it is located in here

`/usr/bin/rtl_433`

it is recomended to uninstall the distro version first. and when you will follow the instruction and compile it by your self. 
After make install rtl_433 is located at here

`/usr/local/bin/rtl_433`

Best regards.

**Comment by joevk5ei** [comment]
Thanks, I will do that.
